Makefile

```diff
@@ -310,20 +310,20 @@ release: clean linux-release macos-release-binary windows-release-binary check
 linux-release: clean lint linux-release-binary embed_crc_helpers gen_release_info
 	mkdir $(RELEASE_DIR)
 
-	@mkdir -p $(BUILD_DIR)/crc-linux-$(CRC_VERSION)-amd64
-	@cp LICENSE $(BUILD_DIR)/linux-amd64/crc $(BUILD_DIR)/crc-linux-$(CRC_VERSION)-amd64
-	tar cJSf $(RELEASE_DIR)/crc-linux-amd64.tar.xz -C $(BUILD_DIR) crc-linux-$(CRC_VERSION)-amd64 --owner=0 --group=0
+	@mkdir -p $(BUILD_DIR)/crc-linux-$(CRC_VERSION)-${GOARCH}
+	@cp LICENSE $(BUILD_DIR)/linux-amd64/crc $(BUILD_DIR)/crc-linux-$(CRC_VERSION)-${GOARCH}
+	tar cJSf $(RELEASE_DIR)/crc-linux-${GOARCH}.tar.xz -C $(BUILD_DIR) crc-linux-$(CRC_VERSION)-${GOARCH} --owner=0 --group=0
 
 	@cp $(RELEASE_INFO) $(RELEASE_DIR)/$(RELEASE_INFO)
 
 	cd $(RELEASE_DIR) && sha256sum * > sha256sum.txt
 
 .PHONY: embed_crc_helpers
-embed_crc_helpers: $(BUILD_DIR)/linux-amd64/crc $(HOST_BUILD_DIR)/crc-embedder
+embed_crc_helpers: $(BUILD_DIR)/linux-${GOARCH}/crc $(HOST_BUILD_DIR)/crc-embedder
 ifeq ($(CUSTOM_EMBED),false)
-	$(HOST_BUILD_DIR)/crc-embedder embed --log-level debug --goos=linux $(BUILD_DIR)/linux-amd64/crc
+	$(HOST_BUILD_DIR)/crc-embedder embed --log-level debug --goos=linux $(BUILD_DIR)/linux-${GOARCH}/crc
 else
-	$(HOST_BUILD_DIR)/crc-embedder embed --log-level debug --cache-dir=$(EMBED_DOWNLOAD_DIR) --no-download --goos=linux $(BUILD_DIR)/linux-amd64/crc
+	$(HOST_BUILD_DIR)/crc-embedder embed --log-level debug --cache-dir=$(EMBED_DOWNLOAD_DIR) --no-download --goos=linux $(BUILD_DIR)/linux-${GOARCH}/crc
```

Comment:
Have you checked that after the naming changes for crc-admin-helper and crc-driver-libvirt, the embedding process still works fine, and that `crc setup` is still able to find/use the embedded files (and not download them) if they are not cached locally?
It looks like this should work, but I may have missed something.

Reply:
I will check and let you know.

Reply:
Tested and it works.
```
$ make embed_crc_helpers 
go build --tags="build" -ldflags="-X github.com/crc-org/crc/v2/pkg/crc/version.crcVersion=2.37.0 -X github.com/crc-org/crc/v2/pkg/crc/version.ocpVersion=4.15.17 -X github.com/crc-org/crc/v2/pkg/crc/version.okdVersion=4.15.0-0.okd-2024-02-23-163410 -X github.com/crc-org/crc/v2/pkg/crc/version.microshiftVersion=4.15.17 -X github.com/crc-org/crc/v2/pkg/crc/version.commitSha=b0972c " -o out/linux-amd64/crc-embedder  ./cmd/crc-embedder
out/linux-amd64/crc-embedder embed --log-level debug --goos=linux out/linux-amd64/crc
DEBU Downloading https://github.com/crc-org/machine-driver-libvirt/releases/download/0.13.8/crc-driver-libvirt-amd64 to /tmp/crc-embedder647635193 
DEBU Download saved to /tmp/crc-embedder647635193/crc-driver-libvirt-amd64 
DEBU Downloading https://github.com/crc-org/admin-helper/releases/download/v0.5.4/crc-admin-helper-linux-amd64 to /tmp/crc-embedder647635193 
DEBU Download saved to /tmp/crc-embedder647635193/crc-admin-helper-linux-amd64 
DEBU Embedding /tmp/crc-embedder647635193/crc-driver-libvirt-amd64 in out/linux-amd64/crc 
DEBU Embedding /tmp/crc-embedder647635193/crc-admin-helper-linux-amd64 in out/linux-amd64/crc 
```

and then
```
$ ./crc setup --log-level debug
DEBU CRC version: 2.37.0+b0972c                   
DEBU OpenShift version: 4.15.17                   
DEBU Running 'crc setup'                          
INFO Using bundle path /home/prkumar/.crc/cache/crc_libvirt_4.15.17_amd64.crcbundle 
DEBU resolv.conf file path: /etc/resolv.conf      
INFO Checking if running as non-root              
INFO Checking if running inside WSL2              
INFO Checking if crc-admin-helper executable is cached 
DEBU crc-admin-helper executable is not cached    
INFO Caching crc-admin-helper executable          
DEBU Trying to extract crc-admin-helper-linux-amd64 from crc executable 
DEBU Extracting embedded 'crc-admin-helper-linux-amd64' from /home/prkumar/work/github/crc/out/linux-amd64/crc to /tmp/crc342705720/crc-admin-helper-linux-amd64 
DEBU Copying '/tmp/crc342705720/crc-admin-helper-linux-amd64' to '/home/prkumar/.crc/bin/crc-admin-helper-linux-amd64' 
DEBU crc-admin-helper executable cached           
DEBU Making /home/prkumar/.crc/bin/crc-admin-helper-linux-amd64 suid 
INFO Using root access: Changing ownership of /home/prkumar/.crc/bin/crc-admin-helper-linux-amd64 
DEBU Running '/usr/bin/sudo chown root /home/prkumar/.crc/bin/crc-admin-helper-linux-amd64' 

```